zenodo/modules/deposit/templates/zenodo_deposit/edit.html

```diff
@@ -38,189 +38,15 @@
-                }
-                $rootScope.$broadcast(
-                  'invenio.records.endpoints.updated', resp.data.links);
-              }, function error(resp) {
-                $rootScope.$broadcast('invenio.records.alert', {
-                  type: 'danger',
-                  data: resp.data,
-                });
-              })
-              .finally(function() {
-                $rootScope.$broadcast('invenio.records.loading.stop');
-              });
-            }
-          };
-        }
-
-        return {
-          scope: false,
-          restrict: 'A',
-          require: '^invenioRecords',
-          link: link,
-        };
-      }]);
-
-      // Overriding ui-select's controller to handle arrays
-      //
-      // Note: UI-Select by default doesn't play well while being inside an
-      // array. What happens is that if the array gets modified from the
-      // outside (eg. delete an array element), the changes are not picked up
-      // by UI-Select, and thus aren't refelcted on its internal model and
-      // view. This leads to inconsistent view state for the end user, eg.
-      // an item from the middle of a ui-select list gets removed using the
-      // 'X' button, but the view will still display it and remove the last
-      // item of the list.
-      //
-      // The remedy for this issue is to handle the update of ui-select's
-      // model manually, by overriding its controller.
-      angular.module('schemaForm')
-        .controller('invenioDynamicSelectController', ['$scope', '$controller',
-          function ($scope, $controller) {
-            $controller('dynamicSelectController', {$scope: $scope});
-            // If it is ui-select inside an array...
-            if ($scope.modelArray) {
-              $scope.$watchCollection('modelArray', function(newValue) {
-                // If this is not the initial setting of the element...
-                if (!angular.equals($scope.select_model, {})) {
-                  // Get the element's correct value from the array model
-                  var value = $scope.modelArray[$scope.arrayIndex][$scope.form.key.slice(-1)[0]];
-                  // Set ui-select's model to the correct value if needed
-                  if ($scope.insideModel !== value) {
-                    $scope.insideModel = value;
-                    var query = $scope.$eval($scope.form.options.processQuery || 'query', {query: value});
-                    $scope.populateTitleMap($scope.form, query);
-                    $scope.select_model.selected = $scope.find_in_titleMap(value);
-                  }
-                }
-              });
-            }
+      // Configure OpenAIRE types and communities
+      app.config(['openAIREProvider', function(openAIREProvider){
+        // TODO: Change `current_openaire.openaire_subtypes`
+        openAIREProvider.setTypes({{current_openaire.openaire_subtypes.openaire_types|tojson}});
```

Comment:
this needs to be changed to plain `current_openaire.openaire_subtypes` in case the property changes